docs/Documentation/Applications/chemicalKinetics.md

```diff
@@ -0,0 +1,284 @@
+	
+	👎 Fewer features than Cantera & Chemkin, C++ only. 
+
+## Installation and testing on Kestrel
+
+### Cantera
+!!! note
+	Cantera can be [installed from source](https://cantera.org/install/compiling-install.html#sec-compiling) or with the conda envionment on the Kestrel as explained below.
+	The performance can vary depending on the compile options and flags while compiling from source. We are more than happy to learn from power users about the flags which lead to the best performance. 
+[Installation: Python version](https://cantera.org/install/conda-install.html#sec-install-conda)
+```
+$ module load conda
+$ conda create --name ct-env --channel cantera cantera ipython matplotlib jupyter
+$ conda activate ct-env
+
+$ python3
+>>> import cantera as ct
+>>> ct.one_atm
+101325.0
+>>> exit() 
+$ conda deactivate
+```
+??? example "[Example interactive jupyter usage](https://nrel.github.io/HPC/Documentation/Development/Jupyter/)"
+	```
+	$ ssh username@kestrel.hpc.nrel.gov
+	```
+	To access your scratch from the jupyter notebook, execute the following from your Kestrel home directory (optional)
+	```
+	$ ln -s /scratch/username scratch
+	```
+	```
+	$ module load conda
+	$ conda activate ct-env
+	```
+	Create a jupyter kernel from ct-env
+	```
+	$ python -m ipykernel install --user --name=ct-env
+	```
+	In a browser, go to [Kestrel JupyterHub](https://kestrel-jhub.hpc.nrel.gov/), select “ct-env” in the Notebook section to open a new jupyter notebook with the ‘ct-env’ loaded
+
+	Try Cantera python API within the notebook, for example,
+	```
+	import cantera as ct
+	ct.one_atm
+	```
+
+[Installation: C++ version](https://cantera.org/install/conda-install.html#sec-conda-development-interface)
+```
+$ module load conda
+$ conda create --name ct-dev --channel cantera libcantera-devel
+$ conda activate ct-dev
+$ conda install cmake scons pkg-config
+
+$ cd /home/username/.conda-envs/ct-dev/share/cantera/samples/cxx/demo
+$ scons && ./demo
+$ cmake . && cmake --build . && ./demo
+$ g++ demo.cpp -o demo $(pkg-config --cflags --libs cantera) && ./demo
+```
+
+??? example "[Example interactive C++ usage](https://www.nrel.gov/hpc/running-jobs.html)"
+	```
+	$ ssh username@kestrel.hpc.nrel.gov
+	```
+	Allocate resources
+	```
+	salloc --account=allocationName --time=00:30:00 --nodes=1 --ntasks-per-core=1 --ntasks-per-node=104 --cpus-per-task=1 --partition=debug
+	```
+	Load Cantera
+	```
+	module load conda
+	conda activate ct-dev
+	```
+	Compile your code
+	```
+	CC -DEIGEN_USE_LAPACKE -DEIGEN_USE_BLAS -fopenmp -O3 -march=native -std=c++17 -I /home/username/.conda-envs/ct-dev/include/cantera/ext -I . mainYourCode.C $(pkg-config --cflags --libs cantera) -o flame.out
+	```
+	Execute
+	``` 
+	srun -n 5 ./flame.out
+	```
+	Please refer to the [job submission documentation](https://www.nrel.gov/hpc/running-jobs.html) for larger jobs in Batch mode.   
```

Comment:
I think it'd be better to link to the gh-pages Slurm documentation section since it has more information